core/app/models/spree/option_value.rb

```diff
@@ -61,6 +61,15 @@ class OptionValue < Spree.base_class
 
     delegate :name, :presentation, to: :option_type, prefix: true, allow_nil: true
 
+    def self.to_tom_select_json
+      all.map do |option_value|
```

Comment:
```suggestion
      all.pluck(:id, :presentation)
```
We don't have to load the entire `ActiveRecord` for each option this way